api/spec/v1.yaml

```diff
@@ -2040,6 +2040,16 @@ components:
           description: |
             The proposal to cancel, if this proposal proposes
             cancelling an existing proposal.
+        parameters_change_module:
+          type: string
+          description: |
+            The name of the module whose parameters are to be changed
+            by this 'parameters_change' proposal.
+        parameters_change:
+          type: string
```

Comment:
It would be preferable to also have `format: byte` here.
Examples:
https://github.com/oasisprotocol/nexus/blob/6b2fe2e4d580e7f9be194081684c19dcf7f1a6bf/api/spec/v1.yaml#L152-L152
https://github.com/oasisprotocol/nexus/blob/6b2fe2e4d580e7f9be194081684c19dcf7f1a6bf/api/spec/v1.yaml#L2295-L2303

Then our openapi generator will represent it as a simple `[]byte` on the Go side, which is more truthful. And when serializing to JSON, it will automatically base64 encode it because that's what the openapi standard prescribes. So you'll be able to drop the ugly manual transformation (`p.ParametersChange = common.Ptr(base64.StdEncoding.EncodeToString(parametersChange))` and related lines).

Reply:
thank you! oh alternatively, I like the JSON solution (that you mention in the PR description) also

Reply:
Used the raw cbor message (bytes) for now.